containerbuddy/config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "consul": "consul:8500",
+  "consul": "{{.CONSUL}}:8500",
+  "onStart": "/opt/containerbuddy/onstart.sh",
   "services": [
     {
-      "name": "wordpress",
+      "name": "app",
```

Comment:
right! forgot about that. I'll make this change, I wasn't very happy with having to change it from "wordpress" and was planning on asking about this.